pwnlib/util/packing.py

```diff
@@ -571,7 +571,7 @@ def _flat(args, preprocessor, packer, filler):
 
 @LocalContext
 def flat(*args, **kwargs):
-    """flat(*args, preprocessor = None, length = None, filler = de_bruijn(),
+    """flat(\*args, preprocessor = None, length = None, filler = de_bruijn(),
```

Comment:
If you introduce backslashes in the docstrings, you should also change the string to raw.
```suggestion
    r"""flat(\*args, preprocessor = None, length = None, filler = de_bruijn(),
```

Reply:
Changed as requested.